lib/npm.js

```diff
@@ -90,7 +90,12 @@ NPM.prototype.listDependencies = function listDependencies(obj, name, prefix) {
         };
     }
 
-    var fileName = name + '-' + newObj.version + '.tgz';
+    // npm pack will remove @ symbols from the name.
+    var santitizedName = name.replace(/[@]/, '');
+    // slashes are converted into dashes by npm pack
+    santitizedName = santitizedName.replace(/[/]/, '-');
```

Comment:
Same question
why not use /\// ?
While granted in this case it might be argued as more readable as /[/]/